src/WPF/ArcGISRuntime.WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -424,5 +456,45 @@ private void ResetCategories()
             // Set the expanded categories.
             SetExpandedCategories(expandedCategoryNames);
         }
+
+        private void ScreenshotButton_Click(object sender, RoutedEventArgs e)
+        {
+            GetJpgImage(SampleContainer);
+        }
+
+        private void GetJpgImage(UIElement source)
+        {
+            ContentControl sampleContainer = (ContentControl)source;
+            int actualHeight = (int)sampleContainer.ActualHeight;
+            int actualWidth = (int)sampleContainer.ActualWidth;
+
+            PresentationSource presentationSource = PresentationSource.FromVisual(source);
+
+            double scaleX, scaleY = 1;
+
+            scaleX = presentationSource.CompositionTarget.TransformToDevice.M11;
+            scaleY = presentationSource.CompositionTarget.TransformToDevice.M22;
+
+            int Height = (int)(source.DesiredSize.Height * scaleY);
+            int Width = (int)(source.DesiredSize.Width * scaleX);
+            System.Windows.Point relativePoint = source.PointToScreen(new System.Windows.Point(0d, 0d));
+            int X = (int)relativePoint.X;
+            int Y = (int)relativePoint.Y;
+            Bitmap screenshot = new Bitmap(Width, Height);
+            Graphics G = Graphics.FromImage(screenshot);
+            G.CopyFromScreen(X, Y, 0, 0, new System.Drawing.Size(Width, Height), CopyPixelOperation.SourceCopy);
```

Comment:
```suggestion
            G.CopyFromScreen((int)relativePoint.X, (int)relativePoint.Y, 0, 0, new System.Drawing.Size(Width, Height), CopyPixelOperation.SourceCopy);
```
X and Y only get used once, we can remove them.